### ISSUE TITLE
Unittest:test_bundle:TestExchangeBundle.test_ingest_candles

### ISSUE THREAD

**Issue by inevity** [enigmampc/catalyst#117]
When set start_dt,end_dt 2017-12-21 22:20:00+00:00 2017-12-22 00:00:00+00:00,
the  got candles from bitfinex alway rangs from **2017-12-23** 13:33:00+00:00 to **2017-12-23** 15:14:00+00:00 .
and the wanted range diff with the got range.

The get_candles calls
```
 self.api.fetch_ohlcv(
                symbol=symbols[0],
                timeframe=timeframe,
                since=ms,
                limit=bar_count,
                params={}
```
which looks like right?!

**Comment by fredfortier** [comment]
I'm trying to understand this issue. Are you saying that Bitfinex does not honor the `since` parameter and gives you the latest bars instead?

**Comment by inevity** [comment]
Yes .
The candles should be in the range from start_dt to end_dt. But When transform to since parameter  , got unexpected candles range. I can confirmation that the since computed correct. and the fetch_ohlcv and bitfinex API  para also correct.

**Comment by fredfortier** [comment]
I think that this is resolved. The last CCXT upgrade fixed this issue with Bitfinex. We included it in 0.4.6. 

We've also created a test suite to run these tests against arbitrary exchanges and assets. See: exchange/test_suites/test_suite_exchange.py:112 and exchange/test_suites/test_suite_bundle.py:109. Expect more refinements on this and unit tests in general in the coming weeks.
